public/scripts/extensions/caption/settings.html

```diff
@@ -54,36 +54,36 @@
                         <option data-type="openrouter" value="google/gemini-pro-vision">google/gemini-pro-vision</option>
                         <option data-type="openrouter" value="google/gemini-flash-1.5">google/gemini-flash-1.5</option>
                         <option data-type="openrouter" value="liuhaotian/llava-yi-34b">liuhaotian/llava-yi-34b</option>
-                        <option data-type="ollama" value="ollama_current">[Currently selected]</option>
+                        <option data-type="ollama" value="ollama_current" data-i18n="[Currently selected]">[Currently selected]</option>
                         <option data-type="ollama" value="bakllava:latest">bakllava:latest</option>
                         <option data-type="ollama" value="llava:latest">llava:latest</option>
-                        <option data-type="llamacpp" value="llamacpp_current">[Currently loaded]</option>
-                        <option data-type="ooba" value="ooba_current">[Currently loaded]</option>
-                        <option data-type="koboldcpp" value="koboldcpp_current">[Currently loaded]</option>
-                        <option data-type="custom" value="custom_current">[Currently selected]</option>
+                        <option data-type="llamacpp" value="llamacpp_current" data-i18n="[Currently loaded]">[Currently loaded]</option>
+                        <option data-type="ooba" value="ooba_current" data-i18n="[Currently loaded]">[Currently loaded]</option>
+                        <option data-type="koboldcpp" value="koboldcpp_current" data-i18n="[Currently loaded]">[Currently loaded]</option>
+                        <option data-type="custom" value="custom_current" data-i18n="[Currently selected]">[Currently selected]</option>
                     </select>
                 </div>
                 <label data-type="openai,anthropic,google" class="checkbox_label flexBasis100p" for="caption_allow_reverse_proxy" title="Allow using reverse proxy if defined and valid.">
                     <input id="caption_allow_reverse_proxy" type="checkbox" class="checkbox">
-                    Allow reverse proxy
+                    <div data-i18n="Allow Reverse Proxy">Allow Reverse Proxy</div>
```

Comment:
Wrong capitalization

Reply:
fixed in https://github.com/SillyTavern/SillyTavern/pull/2376/commits/12c5466f08555ea88e350ba5aff0572878e44ba7